src/main/java/frc/robot/subsystems/ClimbSubsystem.java

```diff
@@ -67,55 +71,103 @@ public ClimbSubsystem(Joystick joystick) {
-    sbclimbHeightOne =
-        climberTab.add("Climb Hight 1", 0).withSize(2, 2).withPosition(8, 0).getEntry();
+    sbClimbOneD =
+        climbTab
+            .add("climb One D", Constants.climbRightPID[2])
+            .withSize(2, 1)
+            .withPosition(8, 0)
+            .getEntry();
+    sbclimbspeedOne =
+        climbTab.add("Climb Current Speed 1", 0).withSize(2, 1).withPosition(10, 0).getEntry();
+    sbclimbheightOne =
+        climbTab.add("Climb targetted height 1", 0).withSize(2, 1).withPosition(12, 0).getEntry();
+
+    // Row 2
+    sbclimbpositionTwo =
+        climbTab.add("climb position 2", 0).withSize(2, 1).withPosition(0, 1).getEntry();
     sbclimbHeightTwo =
-        climberTab.add("Climb Hight 2", 0).withSize(2, 2).withPosition(8, 0).getEntry();
+        climbTab.add("Climb Hight 2", 0).withSize(2, 1).withPosition(2, 1).getEntry();
+    sbClimbTwoP =
+        climbTab
+            .add("climb Two P", Constants.climbLeftPID[0])
+            .withSize(2, 1)
+            .withPosition(4, 1)
+            .getEntry();
+    sbClimbTwoI =
+        climbTab
+            .add("climb Two I", Constants.climbLeftPID[1])
+            .withSize(2, 1)
+            .withPosition(6, 1)
+            .getEntry();
+    sbClimbTwoD =
+        climbTab
+            .add("climb Two D", Constants.climbLeftPID[2])
+            .withSize(2, 1)
+            .withPosition(8, 1)
+            .getEntry();
+    sbclimbspeedTwo =
+        climbTab.add("climb Current Speed 2", 0).withSize(2, 1).withPosition(10, 1).getEntry();
+    sbclimbheightTwo =
+        climbTab.add("Climb targetted height 2", 0).withSize(2, 1).withPosition(12, 1).getEntry();
 
-    sbclimberheightTwo =
-        climberTab
-            .add("Climber targetted height 2", 0)
+    // Row 3 & 4
+    sbClimbEnabbled =
+        climbTab.add("Climb Eanbled", false).withSize(2, 2).withPosition(5, 2).getEntry();
+    sbclimbMode =
+        climbTab
+            .add("Manual Mode Enable", false)
             .withSize(2, 2)
-            .withPosition(8, 0)
+            .withPosition(7, 2)
+            .withWidget(BuiltInWidgets.kToggleSwitch)
             .getEntry();
-    sbclimberspeedTwo =
-        climberTab.add("Climber Current Speed 2", 0).withSize(2, 2).withPosition(0, 2).getEntry();
+    sbclimbSpeedInput =
+        climbTab.add("Climb Speed input", 0.1).withSize(2, 1).withPosition(6, 4).getEntry();
 
-    sbclimberpositionTwo =
-        climberTab.add("Climber position 2", 0).withSize(2, 2).withPosition(2, 4).getEntry();
+    m_climbMotorControllerOne.getPIDCtrl().setP(sbClimbOneP.getDouble(0));
+    m_climbMotorControllerOne.getPIDCtrl().setI(sbClimbOneI.getDouble(0));
+    m_climbMotorControllerOne.getPIDCtrl().setD(sbClimbOneD.getDouble(0));
+
+    m_climbMotorControllerTwo.getPIDCtrl().setP(sbClimbTwoP.getDouble(0));
+    m_climbMotorControllerTwo.getPIDCtrl().setI(sbClimbTwoI.getDouble(0));
+    m_climbMotorControllerTwo.getPIDCtrl().setD(sbClimbTwoD.getDouble(0));
```

Comment:
Unnecessary since MotorController sets the defaults now, and updating is already setup in periodic

Reply:
Fixed that (I blame Jim for that one)